app/jobs/clean_stale_uphold_data_job.rb

```diff
@@ -3,29 +3,29 @@ class CleanStaleUpholdDataJob < ApplicationJob
 
   def perform
     require "sentry-raven"
-    # clear uphold codes sitting for over 5 minutes   
-    publishers = Publisher.has_stale_uphold_code
+    # clear uphold codes sitting for over 5 minutes
+    connections = UpholdConnection.has_stale_uphold_code
     n = 0
-    publishers.each do |publisher|
-      raise if publisher.uphold_status != :code_acquired
-      publisher.uphold_code = nil
-      publisher.save!
+    connections.each do |connection|
+      raise if connection.uphold_status != :code_acquired
+      uphold_code = nil
```

Comment:
I believe this should be `connection.uphold_code = nil`

Reply:
https://github.com/brave-intl/publishers/pull/1825/commits/2717f1080c4c35b859180a970396fb0c9702be5a